### DISCUSSION TITLE
Types for windows library in [@types/react-native-sqlite-storage] is missing

### DISCUSSION THREAD

**Discussion by Hardikb19** [DefinitelyTyped/DefinitelyTyped#67859] [Issues with a @types package]
Can someone add types for windows on @types/react-native-sqlite-storage as well, I generally do not prefer using ts-ignore on my code. This addition would be greatly appreciated

[a comment by an automated account is not shown]

**Comment by dryganets** [comment]
@Hardikb19 , feel free to contribute to the types.
I'm not working nor on react-native nor on windows, so I just have no environment to test this nor I have spare few hours to configure the environment.

I could help you with approvals.
